src/Upgrader/Business/CommandExecutor/AbstractCommandExecutor.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\CommandExecutor;
+
+use Exception;
+use Symfony\Component\Process\Process;
+
+abstract class AbstractCommandExecutor implements CommandExecutorInterface
+{
+    private const DEFAULT_COMMAND_TIMEOUT = 60;
+    private const ERROR_DESCRIPTION = 'Command: [%s]' . PHP_EOL . 'Error: %s';
+
+    /**
+     * @var \Upgrader\Business\CommandExecutor\CommandResultBuilder
+     */
+    private $resultBuilder;
+
+    public function __construct()
+    {
+        $this->resultBuilder = new CommandResultBuilder();
+    }
+
+    /**
+     * @return string
+     */
+    abstract public function getCommand(): string;
+
+    /**
+     * @return int
+     */
+    public function getRequestTimeout(): int
+    {
+        return self::DEFAULT_COMMAND_TIMEOUT;
+    }
+
+    /**
+     * @param string|null $command
+     *
+     * @throws \Exception
+     *
+     * @return \Upgrader\Business\CommandExecutor\CommandResultDto
+     */
+    public function execSuccess(?string $command = null): CommandResultDto
```

Comment:
Why do we have a separate method?
Can we use one method exec instead for exec and execSuccess?

Reply:
Removed this method, still is only "exec"